opm/simulators/flow/MixingRateControls.cpp

```diff
@@ -218,12 +249,12 @@ maxGasDissolutionFactor(const unsigned timeIdx,
                         const int episodeIdx,
                         const int pvtRegionIdx) const
 {
-    if (!this->drsdtActive(episodeIdx) || maxDRs_[pvtRegionIdx] < 0.0) {
+    if (!this->drsdtActive(episodeIdx, pvtRegionIdx) || maxDRs_[pvtRegionIdx] < 0.0) {
```

Comment:
from the code above, 
```c++
    for (std::size_t pvtRegionIdx = 0; pvtRegionIdx < maxDRs_.size(); ++pvtRegionIdx) {
        if (this->drsdtActive(episodeIdx, pvtRegionIdx)) {
            maxDRs_[pvtRegionIdx] = oilVaporizationControl.getMaxDRSDT(pvtRegionIdx) * timeStepSize;
        }
    }
```

Will it be possible to have `this->drsdtActive(episodeIdx, pvtRegionIdx)` is true while `maxDRs_[pvtRegionIdx] < 0.0`?

Reply:
No, it shouldn't. I will double check and remove it if possible.